### PR TITLE
Add tip and background to menu button

### PR DESCRIPTION
**Pull request by onlymash**
Improve layout, add tip and background to menu button

### CODE REVIEW

**Comment by CarlosEsco** [comment]
Screen?

**Comment by onlymash** [comment]
@CarlosEsco 
Something like this(click show background; long click show tip)
Screenshot_20200220-203923~2

File: app/src/main/java/eu/kanade/tachiyomi/ui/recent_updates/RecentChapterHolder.kt [hunk 1]
```diff
@@ -64,7 +68,7 @@ class RecentChapterHolder(private val view: View, private val adapter: RecentCha
         manga_title.text = item.manga.title
 
         // Set the correct drawable for dropdown and update the tint to match theme.
-        chapter_menu_icon.setVectorCompat(R.drawable.ic_more_horiz_black_24dp, view.context.getResourceColor(R.attr.icon_color))
+        chapter_menu.setVectorCompat(R.drawable.ic_more_vert_black_24dp, view.context.getResourceColor(R.attr.icon_color))
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent_updates/RecentChapterHolder.kt]: I don't like that this pushes down the download badges.

**Comment by arkon** [comment]
Took parts of this out into https://github.com/inorichi/tachiyomi/commit/ac3823e10af746ecc19a878fadd79ecf58fb878f

**Pull request closed without merging**